packages/boot/src/__tests__/acceptance/configuration.booter.acceptance.ts

```diff
@@ -0,0 +1,173 @@
+// Copyright IBM Corp. 2019,2020. All Rights Reserved.
+// Node module: @loopback/boot
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  Application,
+  ContextTags,
+  createBindingFromClass,
+  TagValueMatcher,
+} from '@loopback/core';
+import {expect, TestSandbox} from '@loopback/testlab';
+import {Suite} from 'mocha';
+import {resolve} from 'path';
+import {ConfigurationLoader, configurationLoader} from '../../booters';
```

Comment:
`ConfigurationLoader` repeated.

Reply:
No, they diff in case. `ConfigurationLoader` is the type while `configurationLoader` is a decorator function.